### DISCUSSION TITLE
Folder location for Flux models

### DISCUSSION THREAD

**Discussion by nothingness6** [LykosAI/StabilityMatrix#1065] [Q&A]
When you download Flux models from SM, they are placed in the Models/unet folder. However, if you download workflows, some of them require the checkpoint folder to be located in the Models/StableDiffusion folder. As a result, some workflows don't work properly.

What should I do in this case? I'm currently moving the models depending on the workflow each time, and it's becoming quite annoying.

Is there a way to resolve this?

**Comment by mohnjiles** [comment]
I answered your other post: https://github.com/LykosAI/StabilityMatrix/discussions/1060#discussioncomment-11641523

The additional option in the download location selector has been added as of v2.13.0 as well.
